### PR TITLE
Add RBAC support into openstack-networks.yml

### PR DESCRIPTION
**Pull request by MoteHue**

### CODE REVIEW

File: ansible/openstack-networks.yml [hunk 1]
```diff
@@ -12,3 +12,4 @@
       os_networks: "{{ openstack_networks }}"
       os_networks_routers: "{{ openstack_routers }}"
       os_networks_security_groups: "{{ openstack_security_groups }}"
+      os_networks_rbac: "{{ openstack_networks_rbac }}"
```
Comment (markgoddard) [inline ansible/openstack-networks.yml:15]: openstack_networks_rbac needs to be defined.

**Review by markgoddard — CHANGES_REQUESTED** [review]

File: ansible/openstack-networks.yml (hunk 1 above)
Comment (MoteHue) [reply ansible/openstack-networks.yml:15]: Done :+1:

File: ansible/group_vars/all/openstack [hunk 2]
```diff
@@ -21,6 +21,10 @@ openstack_routers: []
 # stackhpc.os-networks role.
 openstack_security_groups: []
 
+# List of RBAC definitions in the openstack projct. Format is as required by the
+# stackhpc.os-networks role.
+openstack_networks_rbac: []
```
Comment (markgoddard) [inline ansible/group_vars/all/openstack:26]: Could you also add it to etc/openstack-config/openstack-config.yml so that the option is visible?

Comment (MoteHue) [reply ansible/group_vars/all/openstack:26]: Done

**Review by markgoddard — APPROVED** [review]
Thanks

**Pull request merged**